.codecov.yml

```diff
@@ -0,0 +1,3 @@
+codecov:
+    notify:
+        after_n_builds: 5
```

Comment:
Where does the 5 come from? The [codecov.io documentation of `after_n_builds`](https://docs.codecov.io/docs/pull-request-comments#section-after_n_builds) says it should be set greater than the number of reports uploaded per commit

Reply:
I read

> Generally, you should set `after_n_builds` equal to the number of reports you upload per commit.

There are five codecov reports (one per job) uploaded per build, so I think this is right. One part I am unclear of is if it refers to number of reports uploaded per *commit* or per *build* - if it is the former, then, were we not to disable codecov on one of two builds per PR, `after_n_builds` should be set to ten. But I think with these changes it might not matter...

Reply:
Will keep an eye out for whether the reported statistics improve - if not we can try bumping from 5 to 10.